README.md

```diff
@@ -1,3 +1,10 @@
+**NOTICE:**
+
+This repository is currently unmaintained. It is recommended to use one of the following plugin managers instead:
```

Comment:
```suggestion
This repository is currently unmaintained. For the time being (as of August, 2023), it is recommended to use one of the following plugin managers instead:
```

Slight tweak to clarify timing, otherwise lgtm, thanks.